app/components/spells.tsx

```diff
@@ -161,11 +204,11 @@ const _SpellList = ({ navigation, database, spells }) => {
             />
         </>
     );
-
 };
-const SpellList = withDatabase(withObservables([], ({ database }) => ({
+
+const SpellList = (['id'], ({ database }) => ({
```

Comment:
This part doesn't look quite right, the `withDatabase` and `withOberservables` function calls should still be in place.

The function calls should look like this:

```
const SpellList = withDatabase(withObservables([], ({ database }) => ({
    spells: database.collections.get('spells').query().observe(),
}))(_SpellList));
```